migrator/src/main/resources/log4j.properties

```diff
@@ -7,11 +7,11 @@
 # LOGGING LEVELS
 
 # To turn more verbose logging on - change "WARN" to "DEBUG"
-log4j.rootLogger=INFO, console
+log4j.rootLogger=DEBUG, console
 
 # LOG FILE LOCATIONS
 log4j.appender.console=org.apache.log4j.ConsoleAppender
-log4j.appender.console.Threshold=INFO
+log4j.appender.console.Threshold=DEBUG

```

Comment:
Will change, i'm stll testing this thing.

On Wed, Jun 3, 2015 at 8:11 AM, Dennis Crissman <notifications@github.com>
wrote:

> In migrator/src/main/resources/log4j.properties
> <https://github.com/lightblue-platform/lightblue-migrator/pull/197#discussion_r31626062>
> :
>
> >
> >  # LOG FILE LOCATIONS
> >  log4j.appender.console=org.apache.log4j.ConsoleAppender
> > -log4j.appender.console.Threshold=INFO
> > +log4j.appender.console.Threshold=DEBUG
>
> debug logging?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-migrator/pull/197/files#r31626062>
> .
>